src/wp-admin/edit-form-blocks.php

```diff
@@ -51,6 +51,8 @@ static function ( $classes ) {
 
 $rest_path = rest_get_route_for_post( $post );
 
+$stylesheet = get_stylesheet();
```

Comment:
Even though it is technically correct to use `$stylesheet` here, I have mixed feelings about this. Part of me wants to rename this to something like `$active_theme`, `$theme_stylesheet`. The return value technically spoken is "The main stylesheet from the active theme or potentially from the child theme is one is used"

So after sharing the thoughts... I still feel that the name is not ideal, but I lack a better solution. In the hope someone else has a better idea, I'll leave this here. But if no one has one, this can be ignored.

Reply:
For what it's worth, I think `$active_theme` is a little clearer, and follows what's already in [site-editor.php](https://github.com/WordPress/wordpress-develop/pull/7661/files#diff-8c4ccab8da60ac2056d717a3517562649b4baab332ca39969445a353d60f64e8R84)